lca-cli/cmd/ibi.go

```diff
@@ -48,7 +50,12 @@ func init() {
 	ibi.Flags().StringVarP(&seedVersion, "seed-version", "", "", "Seed version.")
 	ibi.Flags().StringVarP(&authFile, "authfile", "a", "", "The path to the authentication file of the container registry of seed image.")
 	ibi.Flags().StringVarP(&pullSecretFile, "pullSecretFile", "p", "", "The path to the pull secret file for precache process.")
-
+	ibi.Flags().BoolVarP(&precacheBestEffort, "precache-best-effort", "pb", false, "Set image precache to best effort mode")
```

Comment:
Seems that this flag isn't doing anything

Reply:
This is flag for validation, didn't change logic from IBU
https://github.com/openshift-kni/lifecycle-agent/pull/301/files#diff-441a0b338d8f73e84e4c2fcc41517686d342e7bdd0c567ab582fa909d069bb4fR200